bluemira/equilibria/fem_fixed_boundary/utilities.py

```diff
@@ -350,11 +351,9 @@ def get_flux_surfaces_from_mesh(
             else:
                 index.append(i)
 
-    n = len(index)
-    for xi in range(n):
-        x_1d = np.delete(x_1d, index[xi])
-
-    return x_1d, flux_surfaces
+    mask = np.ones_like(x_1d, dtype=bool)
+    mask[index] = False
+    return x_1d[mask], flux_surfaces
```

Comment:
This was definitely a bug. Rather confusingly, fixing it causes the example to no longer converge.

I think I can explain this behaviour, and potentially fix it, but I require time for further investigation.

@je-cook, separate PR or not?

Reply:
in that case a separate issue and PR is probably the way to go. This now just makes the problem slightly more robust without changing behaviour